core/src/services/oss/backend.rs

```diff
@@ -354,6 +369,7 @@ impl Builder for OssBuilder {
                 server_side_encryption,
                 server_side_encryption_key_id,
                 write_min_size,
+                batch_max_operations,
```

Comment:
Please also set `batch_max_operations` in `Capability`

Reply:
I tried it, but not sure, please take a look